build/DependencyVersions.props

```diff
@@ -37,8 +37,8 @@
     <MicrosoftTemplateEngineCliLocalizationPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineCliLocalizationPackageVersion>
     <MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>
     <MicrosoftTemplateEngineUtilsPackageVersion>$(MicrosoftTemplateEngineCliPackageVersion)</MicrosoftTemplateEngineUtilsPackageVersion>
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0</MicrosoftDotNetPlatformAbstractionsPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>2.1.0</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>DEPENDENCY 'Microsoft.DotNet.PlatformAbstractions' NOT FOUND</MicrosoftDotNetPlatformAbstractionsPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>DEPENDENCY 'Microsoft.Extensions.DependencyModel' NOT FOUND</MicrosoftExtensionsDependencyModelPackageVersion>
```

Comment:
@mmitche do you know where these strings are coming from? (Same problem in https://github.com/dotnet/cli/pull/13740)

Reply:
Huh..not sure. Both should stay at 2.1.0 though